### PR TITLE
Room events: Add some requested aliases

### PR DESCRIPTION
**Pull request by panpawn**

### CODE REVIEW

File: chat-plugins/room-events.js [hunk 1]
```diff
@@ -104,8 +107,9 @@ exports.commands = {
 	},
 	roomeventshelp: [
 		`/roomevents - Displays a list of upcoming room-specific events.`,
-		`/roomevents add [event name] | [event date/time] | [event description] - Adds a room event. Requires: # & ~`,
-		`/roomevents remove [event name] - Deletes an event. Requires: # & ~`,
+		`/roomevents add OR create OR new [event name] | [event date/time] | [event description] - Adds a room event. Requires: # & ~`,
```
Comment (Zarel) [inline chat-plugins/room-events.js]: Don't document aliases!!!!

Comment (Zarel) [reply chat-plugins/room-events.js]: There's only one exception: For commonly-used commands, you can include both the default name of the command, and an abbreviation to make it easier to type.

**Comment by panpawn** [comment]
Updated

File: chat-plugins/room-events.js (hunk 1 above)
Comment (panpawn) [reply chat-plugins/room-events.js]: 👍

**Pull request merged**